control-plane/roles/monitoring/templates/prom_ingress.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
```

Comment:
Decided not to expose prometheus.

Reply:
https://github.com/metal-stack/metal-roles/pull/115/commits/4b6de8a8cece063a4eb8d75a824693fe5b14e952